dags/aps/APSParams.py

```diff
@@ -0,0 +1,21 @@
+class APSParams:
+    def __init__(
+        self,
+        from_date: str = '2000-01-01',
+        until_date: str = '2000-01-02',
```

Comment:
I would put `from_date = today - 1 day` and `until_date = today` instead of hard coded strings